packages/shared/src/features/onboarding/steps/FunnelFact/FunnelFactDefault.tsx

```diff
@@ -4,16 +4,19 @@ import React from 'react';
 import type { ReactElement } from 'react';
 import classNames from 'classnames';
 import Head from 'next/head';
-import { StepHeadline } from '../shared';
-import type { FunnelStepFact } from '../types/funnel';
-import { FunnelStepTransitionType } from '../types/funnel';
-import { FunnelStepCtaWrapper } from '../shared/FunnelStepCtaWrapper';
-import { LazyImage } from '../../../components/LazyImage';
+import { StepHeadline } from '../../shared';
+import type { FunnelStepFact } from '../../types/funnel';
+import { FunnelStepTransitionType } from '../../types/funnel';
+import { FunnelStepCtaWrapper } from '../../shared/FunnelStepCtaWrapper';
+import { LazyImage } from '../../../../components/LazyImage';
 
-const FunnelFact = ({
+export const FunnelFactDefault = ({
   parameters,
   onTransition,
 }: FunnelStepFact): ReactElement => {
+  const isLayoutReversed =
+    parameters?.layout === 'reversed' || parameters?.reverse;
```

Comment:
`parameters` is always defined no? cool if we want to be cautious but just using `?.` everywhere is not a solution.

If we expect parameters to be `undefined` we should type as optional then in `FunnelStepCommon`

Reply:
I haven't dug deeper yet, I just followed how it was structured to avoid any issues. but I applied the optional chaining due to the fact that we render every step and component within a Funnel at all times. We just have those steps with `position: hidden` property.

Reply:
If it is supposed to be optional we can just make it optional in `FunnelStepCommon` type, and thats it